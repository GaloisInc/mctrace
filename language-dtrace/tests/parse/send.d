proc:::exec_success {
  send(12345);
}
