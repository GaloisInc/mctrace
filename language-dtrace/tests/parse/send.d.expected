[Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 0, sourceLine = 1, sourceColumn = 1, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 37, sourceLine = 3, sourceColumn = 1, sourceFile = "tests/parse/send.d"}}, value = TopProbe (Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 0, sourceLine = 1, sourceColumn = 1, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 37, sourceLine = 3, sourceColumn = 1, sourceFile = "tests/parse/send.d"}}, value = Probe {probePatterns = Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 0, sourceLine = 1, sourceColumn = 1, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 18, sourceLine = 1, sourceColumn = 19, sourceFile = "tests/parse/send.d"}}, value = ProbeDescription {probeProvider = "proc", probeModule = "", probeFunction = "", probeName = "exec_success"}} :| [], probeBody = [Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 24, sourceLine = 2, sourceColumn = 3, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 34, sourceLine = 2, sourceColumn = 13, sourceFile = "tests/parse/send.d"}}, value = ExprStmt (Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 24, sourceLine = 2, sourceColumn = 3, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 34, sourceLine = 2, sourceColumn = 13, sourceFile = "tests/parse/send.d"}}, value = Expr {exprApp = Call "send" [Located {sourceRange = SourceRange {sourceFrom = SourcePos {sourceIndex = 29, sourceLine = 2, sourceColumn = 8, sourceFile = "tests/parse/send.d"}, sourceTo = SourcePos {sourceIndex = 33, sourceLine = 2, sourceColumn = 12, sourceFile = "tests/parse/send.d"}}, value = Expr {exprApp = LitInt Decimal 12345}}]}})}]}})}]
